wave_matching/src/icp.cpp

```diff
@@ -13,17 +13,20 @@ ICPMatcher::ICPMatcher(float res) {
     } else {
         this->resolution = -1;
     }
-    wave::ConfigParser parser;
+    ConfigParser parser;
 
     double max_corr = 3, t_eps = 1e-8, fit_eps = 1e-2;
```

Comment:
Yeah that's left over from an earlier stage when these were all hard-coded. I will remove the initialisations.